web/src/components/Tile.tsx

```diff
@@ -51,6 +60,11 @@ const TileContainer = styled.div`
     cursor: pointer;
   }
 
+  & img {
+    filter: ${props =>
+      props.theme.isContrastTheme ? 'invert(1) sepia(0) saturate(0) brightness(20.0) contrast(1.2)' : 'none'};
```

Comment:
<img width="726" alt="image" src="https://github.com/user-attachments/assets/f038f9f7-9543-48fa-83cc-ba80a7d993a7" />

I found the tiles at München to be a bit irregular, I got a more even result with `filter: invert(1) saturate(0) brightness(7)`

Reply:
looks way better :heart_hands: